%YAML:1.0

#--------------------------------------------------------------------------------------------
# Camera Parameters
#--------------------------------------------------------------------------------------------

# Flag do undistortion
flagDoUndistortion: 0 # 0: do not undistort an image (KITTI), 1: undistort an image

# Camera calibration and distortion parameters (OpenCV) 
Camera.fx: 718.856
Camera.fy: 718.856
Camera.cx: 607.1928
Camera.cy: 185.2157

Camera.k1: 0.0
Camera.k2: 0.0
Camera.k3: 0.0
Camera.p1: 0.0
Camera.p2: 0.0

Camera.width: 1241
Camera.height: 376

#--------------------------------------------------------------------------------------------
# User setting Parameters
#--------------------------------------------------------------------------------------------

feature_tracker.thres_error: 100.0      
feature_tracker.thres_bidirection: 1.0
feature_tracker.window_size: 15
feature_tracker.max_level: 6

feature_extractor.n_features: 100
feature_extractor.n_bins_u: 60
feature_extractor.n_bins_v: 30
feature_extractor.thres_fastscore: 40.0
feature_extractor.radius: 5.0

keyframe_update.thres_alive_ratio: 0.7
keyframe_update.thres_mean_parallax: 3.0

map_update.thres_parallax: 1.0
